types/index.d.ts

```diff
@@ -4,11 +4,18 @@ import Vue from 'vue';
 export function install (vue: typeof Vue): void
 
 export interface Tour {
+  // Methods
   start(startStep?: string): void
   previousStep(): void
   nextStep(): void
   stop(): void
   skip(): void
   finish(): void
   currentStep: number
+
+  // Computed
+  isRunning(): boolean
```

Comment:
This is not working. You have to remove `()` for the computed props.
E.g.:
```
isRunning: boolean
```

Reply:
Thanks for the feedback. I've pushed the update. Let me know if that works better.

Reply:
Now it works. I have tested 👍